.yardopts

```diff
@@ -0,0 +1,4 @@
+--markup=markdown
+--load docs/yard_support.rb
+--tag tags:tags
+--tag availability:availability
```

Comment:
It does the same as `PluginParser` does but in the command-line option form.

https://github.com/danger/danger/blob/4f0de68acc84184ca768d1f5cf40301abeaba1c3/lib/danger/plugin_support/plugin_parser.rb#L58-L59